static/app/components/events/breadcrumbs/breadcrumbsDrawer.tsx

```diff
@@ -19,11 +21,16 @@ import {
   BREADCRUMB_SORT_OPTIONS,
   BreadcrumbSort,
 } from 'sentry/components/events/interfaces/breadcrumbs';
+import {DrawerBody, DrawerHeader} from 'sentry/components/globalDrawer/components';
```

Comment:
I don't have a strong opinion on it for this use case specifically, but for other compound components (like Tab, MultipleCheckbox, GuidedSteps) we have named the components like `Drawer.Body` to denote that they need to be used inside of the parent component. It's a bit different here though since you aren't actually rendering `<Drawer />` as the wrapper, so it could go either way for me. Just wanted to point it out

Reply:
I thought about it but I think because there isn't the Drawer wrapper I'll retain this as is. They're entirely optional as well since the drawer renders empty, so I think it's okay as is.